tools/ClassGen/InstrGen.cpp

```diff
@@ -529,6 +529,17 @@ int main(int argc, char **argv) {
   //===--------------------------------------------------------------------===//
   //                Non-linearities
   //===--------------------------------------------------------------------===//
+  BB.newInstr("Relu")
+      .addOperand("Dest", OperandKind::Out)
+      .addOperand("Src", OperandKind::In)
+      .inplaceOperand({
+          "Dest",
+          "Src",
+      })
+      .dataParallel()
+      .autoVerify(VerifyKind::SameElementType, {"Dest", "Src"})
+      .autoVerify(VerifyKind::SameShape, {"Dest", "Src"})
```

Comment:
I *think* you can combine these into one check using `VerifyKind::SameType` but I'm not sure.

Reply:
Perhaps, this matches the style of the surrounding instructions though. Can we do that in a follow up?